implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/DynatraceConfigTest.java

```diff
@@ -15,18 +15,36 @@
  */
 package io.micrometer.dynatrace;
 
+import com.dynatrace.file.util.FileBasedConfigurationTestHelper;
 import com.dynatrace.metric.util.DynatraceMetricApiConstants;
 import io.micrometer.core.instrument.config.validate.InvalidReason;
 import io.micrometer.core.instrument.config.validate.Validated;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 class DynatraceConfigTest {
+    private static String nonExistentConfigFileName = UUID.randomUUID().toString();
+    
+    @BeforeEach
+    void setUp() {
+        // This is required for environments where the 
```

Comment:
So this setup is just really to ensure that all existing tests don't use any config that *might* exist? If that's true, I think you can simplify this a bit with something like:

```
Make sure that all tests get the default configuration, 
even if there's an `endpoint.properties` file in place
```
Does this makes sense?